client/src/components/buttonRender.jsx

```diff
@@ -0,0 +1,144 @@
+let buttonRender = function (curPage, totalPages, isReview, changePage) {
```

Comment:
I think the variable name `isReviewed` is more fit with your code.

Reply:
Thanks, just changed the name to 

> isReviewed`